spec/services/efile/poll_for_acknowledgments_service_spec.rb

```diff
@@ -117,8 +117,9 @@
 
             it "changes the state from transmitted to investigating" do
```

Comment:
This `it "..."` line doesn't seem to describe what happens. What about `it "changes the state to accepted with the imperfect return acceptance flag"`?